apps/ex_wire/lib/ex_wire/handshake.ex

```diff
@@ -143,6 +144,44 @@ defmodule ExWire.Handshake do
     end
   end
 
+  @doc """
+  Builds an AuthMsgV4 (see build_auth_msg/5), serializes it, and encodes it.
+  This message is ready to be sent to a peer to initiate the encrypted handshake.
+  """
+  @spec build_encoded_auth_msg(
+          ExthCrypto.Key.public_key(),
+          ExthCrypto.Key.private_key(),
+          ExthCrypto.Key.public_key(),
+          nonce() | nil,
+          ExthCrypto.Key.key_pair() | nil
+        ) :: {binary(), ExthCrypto.Key.key_pair(), nonce()}
+  def build_encoded_auth_msg(
+        my_static_public_key,
+        my_static_private_key,
+        her_static_public_key,
```

Comment:
I forgot I wanted to update all of these to initiator and recipient as needed. I think it's far more clear, so I'll update those.

Reply:
Updated function names and simplified all of these parameters in da0fa1db209bacfdc535b888c766d8e076e1f0b5